include/zephyr/modem/ubx/protocol.h

```diff
@@ -74,12 +74,19 @@ enum ubx_nav_fix_type {
 	UBX_NAV_FIX_TYPE_TIME_ONLY = 5,
 };
 
+enum ubx_nav_hp_dgnss_mode {
+	UBX_NAV_HP_DGNSS_MODE_RTK_FLOAT = 2,
+	UBX_NAV_HP_DGNSS_MODE_RTK_FIXED = 3,
+};
+
 #define UBX_NAV_PVT_VALID_DATE				BIT(0)
 #define UBX_NAV_PVT_VALID_TIME				BIT(1)
 #define UBX_NAV_PVT_VALID_UTC_TOD			BIT(2)
 #define UBX_NAV_PVT_VALID_MAGN				BIT(3)
 
 #define UBX_NAV_PVT_FLAGS_GNSS_FIX_OK			BIT(0)
+#define UBX_NAV_PVT_FLAGS_GNSS_CARR_SOLN_FLOATING	(BIT(6))
+#define UBX_NAV_PVT_FLAGS_GNSS_CARR_SOLN_FIXED		(BIT(7))
```

Comment:
drop the outer parenthesis, redundant, you only need to parenthesize arguments or operations, adding extra parenthesis randomly everywhere makes this a huge mess when you'll inevitably try to troubleshoot something when looking at the intermediate files